src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="Surname">
+            Surname:
+          </label>
+          <input
+          type="text"
+          name="Surname"
+          id="Surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            id="Surname"
            autocomplete="off"
          >
```
format your code

checklist point
<img width="932" alt="image" src="https://user-images.githubusercontent.com/102411130/229719991-583f9b09-1d13-4c11-9cb9-f84b45885cde.png">